app/main.py

```
@@ -0,0 +1,55 @@
+@app.post("/todos", status_code=201)
+async def create_todo(
+    todo: schemas.ToDoCreate, db: AsyncSession = Depends(get_db)
+) -> schemas.ToDoResponse:
+    db_todo = models.ToDo(**todo.model_dump())  # Error handling done by Pydantic
+
+    return await crud.create_todo(db, db_todo)
+
+
+# Delete a ToDo item
+@app.delete("/todos/{todo_id}", status_code=204)
+async def delete_todo(todo_id: int, db: AsyncSession = Depends(get_db)) -> None:
```

Ruff (0.8.2)

Do not perform function call `Depends` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)